Cart/Cart.css

```diff
@@ -0,0 +1,96 @@
+.cart__page{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+ }
+ .fa-close:hover{
+     background-color: black;
+     color:var(--primary-color);
+     border-radius: 50%;
+     padding:0.2rem 0.5rem;
+ }
+ .cart__payment{
+     margin-top:5rem;
+ }
+ .card__container{
+     margin-top:5rem;
+     grid-area: main;
+     display: flex;
+     flex-direction: column;
+ }
+ 
+ .card__hori2{
+     padding:1rem 4rem ;
+     width:35rem !important;
```

Comment:
Any particular reason to `!important` ? Maybe try solving the problem using specificity.

Reply:
Okay I will try to improve it through specificity